### DISCUSSION TITLE
Automatic "apply plan" on startup not working

### DISCUSSION THREAD

**Discussion by toyoto** [aredden/electron-G14Control#52] [Q&A]
Hello !

G14 is a super nice piece of software, I really enjoy it as a sound recorder!  But I need to apply the Omega Saver plan on each boot, even when I unplugged the AC. The application start on boot, but the plan aren't auto applying. Do I need to change something on the application. If it becomes a suggestion (if the feature isn't existing), I may need to change the category of the discussion into "idea" :-)

Thank you so much for your support.
Best regards

**Comment by toyoto** [comment]
Hello again. Any ideas how to auto start / auto apply plans when starting the computer or when the computer is on but I unplugged / plug an external power supply ?
Thanks

**Reply by aredden** [reply]
Strange that it wouldn't work automatically. You have the "apply on start" box checked? Also, does g14control start on boot?

**Reply by toyoto** [reply]
My bad! The boxes weren't checked... G14control start on boot. Thank you so much!
